module-2/backend/week_one.md

```diff
@@ -4,31 +4,66 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 
 Note: When you're done, submit a PR. 
 
-1. List the five common HTTP verbs and what the purpose is of each verb.
+1. List the five common HTTP verbs and what the purpose is of each verb.  
+  *GET: Go to the path specified  
+  *POST:  Used when making a new object 
+  *PUT: Used for updating an object 
+  *DELETE: Used for deleting and object 
+  *PATCH: Not quite sure what this does   

```

Comment:
Also used for updating a resource. Generally interchangeable with PUT, though some will say that PUT is to completely replace the resource, while PATCH is to update just some attributes.